api/api_public_transaction_pool.go

```diff
@@ -217,6 +218,7 @@ func RpcOutputReceipt(tx *types.Transaction, blockHash common.Hash, blockNumber
 
 	fields["logsBloom"] = receipt.Bloom
 	fields["gasUsed"] = hexutil.Uint64(receipt.GasUsed)
+	fields["effectiveGasPrice"] = hexutil.Uint64(tx.GasPrice().Uint64())
```

Comment:
Don't we need to return baseFee for `effectiveGasPrice` field?

Reply:
https://github.com/bobpkr/klaytn/blob/api-dynamicGasPrice/api/api_public_transaction_pool.go#L268-L273

after function `RpcOutputReceipt` is called, `effectiveGasPrice` field is set by header's baseFee.

Reply:
fixed it by adding block's header to the arguments of`RpcOutputReceipt` function and setting `effectiveGasPrice` field to baseFee when baseFee is not null.